boards/arm/sama5/giant-board/include/board.h

```diff
@@ -0,0 +1,316 @@
+ *
+ * Reset
+ * 3.3V
+ * VREF
+ * GND
+ * AD4 / PD23
+ * AD2 / PD21
+ * AD1 / PD20
+ * AD5 / PD24
+ * AD3 / PD22
+ * AD0 / PD19
+ * SCK / PA14
+ * MOSI / PA15
+ * MISO / PA16
+ * RX / PD2
+ * TX / PD3
+ * GND
+ *
+ * J2 - right pins
+ *
+ * VBAT
+ * EN
+ * VBUS
+ * PD13
+ * PD31
+ * PWM1 / PB0
+ * PWM3 / PB7
+ * PWML1 / PB1
+ * PWM2 / PB5
+ * PB3 / PWMEXTRG
+ * SCL / PC0
+ * SDA / PB31
+ *
+ */
+
+/* J1 - left pins */
+
+#define GB_PIO_AD4          PIO_ADC_AD4         /* AD4 / PD23 */
+#define GB_PIO_PD23         PIO_ADC_AD4         /* AD4 / PD23 */
+
+#define GB_PIO_AD2          PIO_ADC_AD2         /* AD2 / PD21 */
+#define GB_PIO_PD21         PIO_ADC_AD2         /* AD2 / PD21 */
+
+#define GB_PIO_AD1          PIO_ADC_AD1         /* AD1 / PD20 */
+#define GB_PIO_PD20         PIO_ADC_AD1         /* AD1 / PD20 */
+
+#define GB_PIO_AD5          PIO_ADC_AD5         /* AD5 / PD24 */
+#define GB_PIO_PD24         PIO_ADC_AD5         /* AD5 / PD24 */
+
+#define GB_PIO_AD3          PIO_ADC_AD3         /* AD3 / PD22 */
+#define GB_PIO_PD22         PIO_ADC_AD3         /* AD3 / PD22 */
+
+#define GB_PIO_AD0          PIO_ADC_AD0         /* AD0 / PD19 */
+#define GB_PIO_PD19         PIO_ADC_AD0         /* AD0 / PD19 */
+
+#define GB_PIO_SCK          PIO_SPI0_SPCK_1     /* SCK / PA14 */
+#define GB_PIO_PA14         PIO_SPI0_SPCK_1     /* SCK / PA14 */
+
+#define GB_PIO_MOSI         PIO_SPI0_MOSI_1     /* MOSI / PA15 */
+#define GB_PIO_PA15         PIO_SPI0_MOSI_1     /* MOSI / PA15 */
+
+#define GB_PIO_MISO         PIO_SPI0_MISO_1     /* MISO / PA16 */
+#define GB_PIO_PA16         PIO_SPI0_MISO_1     /* MISO / PA16 */
+
+#define GB_PIO_RX           PIO_UART0_TXD       /* RX / PD2 */
+#define GB_PIO_PD2          PIO_UART0_TXD       /* RX / PD2 */
+
+#define GB_PIO_TX           PIO_UART0_RXD       /* TX / PD3 */
+#define GB_PIO_PD3          PIO_UART0_RXD       /* TX / PD3 */
+
+/* J2 - right pins */
+
+#define GB_PIO_PD13         PIO_TC1_CLK_2       /* PD13 */
+
+#define GB_PIO_PD31         PIO_TC3_CLK_3       /* PD31 */
+
+#define GB_PIO_PWM1         PIO_PWM0_H1         /* PWM1 / PB0 */
+#define GB_PIO_PB0          PIO_PWM0_H1         /* PWM1 / PB0 */
+
+#define GB_PIO_PWM3         PIO_PWM0_H3         /* PWM3 / PB7 */
+#define GB_PIO_PB7          PIO_PWM0_H3         /* PWM3 / PB7 */
+
+#define GB_PIO_PWML1        PIO_PWM0_L1         /* PWML1 / PB1 */
+#define GB_PIO_PB1          PIO_PWM0_L1         /* PWML1 / PB1 */
+
+#define GB_PIO_PWM2        PIO_PWM0_H2          /* PWM2 / PB5 */
+#define GB_PIO_PB5         PIO_PWM0_H2          /* PWM2 / PB5 */
+
+#define GB_PIO_PB3         PIO_PWM0_EXTRG0      /* PB3 / PWMEXTRG */
+#define GB_PIO_PWMEXTRG    PIO_PWM0_EXTRG0      /* PB3 / PWMEXTRG */
+
+#define GB_PIO_SCL         PIO_TWI0_CK_2        /* SCL / PC0 */
+#define GB_PIO_PC0         PIO_TWI0_CK_2        /* SCL / PC0 */
+
+#define GB_PIO_SCL        PIO_TWI0_D_2          /* SDA / PB31 */
+#define GB_PIO_PB31       PIO_TWI0_D_2          /* SDA / PB31 */
+
```

Comment:
The above definitions are all internal to the board implementation and must NOT be exposed in the board.h header file.  Please move to the giant-board.h header file.

Reply:
@patacongo will fix.